articles/soil-carbon-comment/index.md

```diff
@@ -172,7 +176,11 @@ But carbon credits aren’t the only way to support soil carbon. An alternative
 
 Meanwhile, any efforts built around carbon offset credits should be transparent, subject to third-party verification, and capable of thorough validation by financially independent organizations.
 
-## Terms of engagement
+## Credits
```

Comment:
I suggest we bump the version number to 1.0.1 for the article and add a section above that reads something like:

```markdown
## Update 8/25/2020

After reviewing an updated version of the draft protocol, we submitted a second <TaggedLink action='PDF' category='downloads' href='https://carbonplan-assets.s3.amazonaws.com/docs/Soil-Carbon-Comment-Letter-08-25-2020.pdf'>comment letter</TaggedLink> raising additional issues with the proposed protocol.
```